packages/waku/src/lib/middleware/rsc/ssr.ts

```diff
@@ -291,16 +291,15 @@ export const renderHtml = async <Context>(
   const moduleMap = new Proxy(
     {},
     {
-      get(_target, filePath: string) {
+      get(_target, _filePath: string) {
```

Comment:
Please don't use `_` prefix other than for "unused parameters".

Reply:
Yes sure, I was just trying to do a prototype. I'll fix it.